package.json

```
@@ -103,6 +104,7 @@
     "tsup": "catalog:repo",
     "turbo": "^2.0.14",
     "turbo-ignore": "^2.0.6",
+    "typedoc": "^0.27.6",
```

[nit] would need to go along with `pnpm install`

```suggestion
    "typedoc": "0.27.6",
```